workspace/utilities/master/master.xsl

```diff
@@ -36,7 +36,7 @@
 	<!-- Buttons -->
 	<xsl:import href="../lib/button.xsl" />
 	<xsl:import href="../lib/button-tel.xsl" />
-	<xsl:import href="../lib/nav-link.xsl" />
+	<xsl:import href="../lib/edit-buttons.xsl" />
```

Comment:
This should be another PR.. or maybe we now have it in duplicate ??

Reply:
@nitriques Fixed!